xdsl/ir/core.py

```diff
@@ -870,10 +878,24 @@ def clone(
         return op
 
     @classmethod
-    def has_trait(cls, trait: type[OpTrait], parameters: Any = None) -> bool:
+    def has_trait(
+        cls,
+        trait: type[OpTrait],
+        parameters: Any = None,
+        allow_unregistered: bool = False,
```

Comment:
```suggestion
        value_if_unregistered: bool = False,
```
I believe this should convey better the meaning of that argument?